recipes/infer-subc/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "infer-subc" %}
+{% set version = "0.0.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/infer-subc-{{ version }}.tar.gz
+  sha256: 7ff6222f124d7167c3334e25d0178adc8d0205b3f5f189fb7ebc4e34379959a8
+
+build:
+  entry_points:
+    - infer_subc = infer_subc.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools-scm
+  run:
+    - aicsimageio
+    - aicssegmentation
+    - jupyter
+    - napari
+    - numpy
+    - pandas
+    - python >=3.8
+    - scikit-image
+    - scipy
+
+test:
+  imports:
+    - infer_subc
+  commands:
+    - pip list
+    # - pip check  # Fails due to aicssegmentation and itkwidgets
+    - infer_subc --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ndcn/infer-subc
+  summary: A plugin that enables organelle segmentation
+  license: Unlicense
```

This is BSD-3 I think? https://github.com/ndcn/infer-subc/blob/main/LICENSE

```suggestion
  license: BSD-3-Clause
```

---

That is not the file that comes with the tar file https://files.pythonhosted.org/packages/68/2b/9c2911b9204e97e949d7c327a5c9f19e2565956acf9582d31b15f0640a2c/infer-subc-0.0.post1.tar.gz